File: Proyecto-Prog_Avanzada-railway/src/main/java/co/edu/uniquindio/proyecto/proyectotienda/dto/UsuarioGetDTO.java
package co.edu.uniquindio.proyecto.proyectotienda.dto;

import lombok.*;

import java.util.List;

@AllArgsConstructor
@Getter
@Setter
@ToString
public class UsuarioGetDTO {

    private int codigoUsuario;

    private String nombre;

    private String apellido;

    private String email;

    private String contrasenia;

    private String direccion;

    private String telefono;

}

File: Proyecto-Prog_Avanzada_Frontend-master/src/app/modelo/usuario-get-dto.ts
export class UsuarioGetDTO {
    codigo: number = 0;
    nombre: string = "";
    apellido: string = "";
    email: string = "";
    contrasenia: string = "";
    direccion: string = "";
    telefono: string = "";

    constructor(codigo: number, nombre: string, apellido: string, email: string, contrasenia: string, direccion: string, telefono: string) {
        this.codigo = codigo;
        this.nombre = nombre;
        this.apellido = apellido;
        this.email = email;
        this.contrasenia = contrasenia;
        this.direccion = direccion;
        this.telefono = telefono;
    }
}
